### PR TITLE
Minor changes to command line parameter parsers.

### PR DESCRIPTION
**Pull request by ashelkovnykov**

### CODE REVIEW

File: photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala [hunk 1]
```diff
@@ -180,8 +180,6 @@ class GameTrainingDriverTest {
       Array(validParamMap.copy.remove(GameTrainingDriver.inputDataDirectories)),
       // No root output directory
       Array(validParamMap.copy.remove(GameTrainingDriver.rootOutputDirectory)),
-      // No feature bags directory
```
Comment (cmjiang) [inline photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala]: Does this change allow user to input an empty feature bag without raising an error? Therefore users don't need to manually change the params at job configurations.

**Comment by jianlingzhong** [comment]
looks good to me.

File: photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala (hunk 1 above)
Comment (ashelkovnykov) [reply photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala]: No, but that's a good point - let me add that change.

Comment (yunboouyang) [reply photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala]: Can you illustrate why you want to delete this test?

Comment (ashelkovnykov) [reply photon-client/src/test/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverTest.scala]: Feature white lists are not a mandatory feature - not having them present will not cause parameter parsing to fail, and thus is not invalid input, which is what this test checks.

**Pull request merged**